basic/ncr.py

```diff
@@ -0,0 +1,16 @@
+def nCr(n, r):
+    return (fact(n) / (fact(r)* fact(n - r)))
+
+
+# Returns factorial of n
+def fact(n):
+    res = 1
+
+    for i in range(2, n + 1):
+        res = res * i
```

Comment:
```suggestion
        res *= i
```

Use augmented assignment for marginally better efficiency / tidiness https://docs.python.org/3/reference/simple_stmts.html#augmented-assignment-statements